api/tools.ts

```
@@ -1,41 +1,77 @@
+import { registerEchoTool } from "./tools/utils/echo";
+import { registerJoinSpeakingTool } from "./tools/bots/join-speaking";
 
 export function registerTools(server: McpServer, apiKey: string): McpServer {
   const baasClient = new BaasClient({
     apiKey: apiKey,
+    baseUrl: "https://api.meetingbaas.com/",
   });
 
   // Register bot tools
-  const updatedServer = registerJoinTool(server, baasClient);
+  let updatedServer = registerJoinSpeakingTool(server);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updatedServer' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)